cmd/edgectl/aes_install_errors.go

```diff
@@ -93,103 +100,30 @@ func (i *Installer) GetVersionsError(err error) Result {
-		URL:          url,
-		Err:          errors.Wrap(err, "Failed to download AES CRD manifests"),
-	}
-}
-
-// Unable to fetch the AES manifests (aes.yaml)
-func (i *Installer) AESManifestsError(err error) Result {
-	i.Report("fail_no_internet", ScoutMeta{"err", err.Error()})
-
-	url := "https://www.getambassador.io/docs/latest/topics/install/help/aes-manifests"
-	return Result{
-		ShortMessage: "The installer failed to download the AES manifests",
-		Message:      fmt.Sprintf("Visit %v for a more detailed explanation and step-by-step instructions about downloading AES manifests to continue installing Ambassador Edge Stack.", url),
+		ShortMessage: "The installer failed to download the AES Chart",
+		Message:      fmt.Sprintf("Visit %v for a more detailed explanation and step-by-step instructions about downloading AES Chart to continue installing Ambassador Edge Stack.", url),
 		URL:          url,
-		Err:          errors.Wrap(err, "Failed to download AES manifests"),
-	}
-}
-
-// Unable to parse the downloaded AES manifests
-func (i *Installer) ManifestParsingError(err error, matches []string) Result {
-	url := "https://www.getambassador.io/docs/latest/topics/install/help/manifest-parsing"
-
-	i.log.Printf("matches is %+v", matches)
-
-	return Result{
-		Report:       "fail_bad_manifests",
-		ShortMessage: "The installer failed to parse AES manifests",
-		Message:      fmt.Sprintf("Visit %v for a more detailed explanation and step-by-step instructions about downloading AES manifests to continue installing Ambassador Edge Stack.", url),
-		URL:          url,
-		Err:          err,
+		Err:          errors.Wrap(err, "Failed to download AES CRD Chart"),
 	}
 }
 
 // Existing AES CRD's of incompatible version
-func (i *Installer) IncompatibleCRDVersionsError(installedVersion string) Result {
-	url := "https://www.getambassador.io/docs/latest/topics/install/help/incompatible-crd-versions"
-	i.Report("fail_existing_aes", ScoutMeta{"installing", i.version}, ScoutMeta{"found", installedVersion})
-
-	abortExisting := `
-This tool does not support upgrades/downgrades at this time.
-The installer will quit to avoid corrupting an existing installation of AES.`
-
-	message := fmt.Sprintf("Visit %v for a more detailed explanation and step-by-step instructions about updating CRD versions to continue installing Ambassador Edge Stack.\n%v", url, abortExisting)
-
-	return Result{
-		URL:          url,
-		ShortMessage: fmt.Sprintf("The installer found incompatible AES CRD versions: Existing AES %v found when installing AES %v", installedVersion, i.version),
-		Message:      message,
-		Err:          errors.Errorf("Ambassador Edge Stack %s already installed", installedVersion),
-	}
-}
-
-// Existing AES CRD's, unable to upgrade.
-func (i *Installer) ExistingCRDsError() Result {
+func (i *Installer) ExistingInstallationFoundError(installedVersion string) Result {
 	url := "https://www.getambassador.io/docs/latest/topics/install/help/existing-crds"
```

Comment:
This needs to be a new help page (with a new URL).

Reply:
Done